kubetest2/kubetest2-kind/deployer/deployer.go

```diff
@@ -74,6 +78,15 @@ func bindFlags(d *deployer) *pflag.FlagSet {
 	flags.StringVar(
 		&d.nodeImage, "build-type", "", "--type for kind build node-image",
```

Comment:
note that this may go away upstream depending on how K/K handles changes to their build we will focus on whatever is going to be supported properly.